src/orders/template.yaml

```diff
@@ -12,10 +12,10 @@ Parameters:
     Type: String
   StageName:
     Type: String
-    Default: dev
+    Default: development
```

Comment:
Why did it change? I think we have dev in 2 lambdas

Reply:
Required for [settings](https://github.com/infinity-cloud-solutions/delivery-planner-backend/blob/ce861c9bc539fce5fd9657a11bbfb28b1c826db7/src/orders/settings.py#L21) to work